pkg/apis/starlingx/v1/system_types_test.go

```diff
@@ -20,6 +20,8 @@ func TestStorageSystem(t *testing.T) {
 	description := string("A sample description")
 	location := string("A sample location")
 	contact := string("A sample contact")
+	latitude := string("A sample latitude")
+	longitude := string("A sample longitude")
```

Comment:
Shouldn't these be somewhat valid latitude/longitude values?

Reply:
I think that there is no difference for testing, but I can set these sample to valid values

Reply:
Thanks. That would serve as a bit of an example, at least, short of adding something under the examples dir